petab/yaml.py

```diff
@@ -159,4 +158,36 @@ def write_yaml(yaml_config: Dict[str, Any], filename: str) -> None:
     """
 
     with open(filename, 'w') as outfile:
-        yaml.dump(yaml_config, outfile, default_flow_style=False)
+        yaml.dump(yaml_config, outfile, default_flow_style=False,
+                  sort_keys=False)
+
+
+def create_default_yaml(sbml_file: str,
```

Comment:
Why 'default'? Maybe something like `{create|write}_problem_yaml`?

Directly allow for lists of filenames where they are allowed in yaml?